### PR TITLE
[JBPM-8496] PIM UI - migration logs are presented as a long text

### PR DESCRIPTION
**Pull request by yzhao583**
Use a table to display the migration log for a migration. User can sort by column, and can view records using the pagination feature. When users click the "View Log" button (a button with the info icon), a modal contains the details of the log should pop up.

Here is example of the table:
image

Here is an example of the details log:
image

### CODE REVIEW

**Comment by kie-ci2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by yzhao583** [comment]
Fix https://issues.jboss.org/browse/JBPM-8496

**Comment by yzhao583** [comment]
Here is example of the table:
image

Here is an example of the details log:
image

File: process-migration-service/frontend/src/component/common/AdvancedTable.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    }
+  };
+
+  onSubmit = () => {
+    this.setPage(this.state.pageChangeValue);
+  };
+  setPage = value => {
+    const page = Number(value);
+    if (
+      !Number.isNaN(value) &&
```
[the remark this answers is not in the document]
Comment (yzhao583) [reply process-migration-service/frontend/src/component/common/AdvancedTable.js]: @jstastny-cz You are right, will update, thanks!

**Comment by yzhao583** [comment]
@ruromero @jstastny-cz Updated! Could you please review again? Thanks

**Comment by jstastny-cz** [comment]
ok to test

**Pull request merged**